lib/logstash/outputs/elasticsearch/template_manager.rb

```diff
@@ -19,9 +19,11 @@ def self.install_template(plugin)
     end
 
     private
-    def self.get_template(path, es_major_version)
-      template_path = path || default_template_path(es_major_version)
+    def self.load_default_template(es_major_version, ecs_compatibility)
+      template_path = default_template_path(es_major_version, ecs_compatibility)
       read_template_file(template_path)
+    rescue => e
+      fail "Failed to load default template for Elasticsearch v#{es_major_version} with ECS #{ecs_compatibility}; caused by: #{e.message}"
```

Comment:
nitpick: not sure what kind of (io) exceptions might go wrong here, we might want `e.inspect` just in case?